server/routes/analyticsRouter.ts

```diff
@@ -185,6 +206,17 @@ export default function routes(router: Router): Router {
 
     const protectedCharacteristic = protectedCharacteristicRoutes[characteristicName].characteristic
 
+    const groupsForCharacteristic = knownGroupsFor(protectedCharacteristic)
```

Comment:
Maybe we need to do something similar to what's done in the service to get rid of the young people value for non-YCS prisons :|

https://github.com/ministryofjustice/hmpps-incentives-ui/blob/main/server/services/analyticsService.ts#L291-L294

Reply:
Yes, and thanks for explaining the logic, updated that